install/installer/pkg/components/ws-manager-bridge/configmap.go

```diff
@@ -16,7 +16,7 @@ import (
 
 func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 	wsmbcfg := Configuration{
-		Installation:                        "",
+		Installation:                        ctx.InstallationShortname,
```

Comment:
Related to above, we should probably use `ctx.Config.Metadata.InstallationShortname`

Reply:
If you change to a pointer, use the `pointer.StringDeref` function. That will convert a pointer to a string and allow you specify a default value

Reply:
Where should that function be used? It only makes sense to set a default in a central location, so I opted for `config.go`.

Reply:
When you've finished lunch, do you want ~10 mins pairing on it? Probably easier than async

Reply:
👍

Reply:
Resolved in code.